package.json

```diff
@@ -27,13 +27,17 @@
   "license": "MIT",
   "dependencies": {
     "bcrypt-nodejs": "0.0.3",
+    "browserify": "^13.0.0",
     "cheerio": "0.20.0",
     "commander": "2.9.0",
     "event-stream": "3.3.2",
     "express": "4.13.4",
+    "favico.js": "^0.3.10",
+    "handlebars": "^2.0.0",
+    "jquery-browserify": "^1.8.1",
     "lodash": "~2.4.1",
     "mkdirp": "0.5.1",
-    "moment": "2.11.2",
+    "moment": "^2.7.0",
```

Comment:
Moment follows semver, so upgrading to 2.11.2 shouldn't break anything. Infact, it was upgraded on the server just a couple of weeks or so ago.